client/components/domains/contact-details-form-fields/custom-form-fieldsets/us-address-fieldset.jsx

```diff
@@ -15,15 +15,28 @@ import { StateSelect, Input } from 'my-sites/domains/components/form';
 
 const UsAddressFieldset = props => {
 	const { getFieldProps, translate, countryCode } = props;
+	const STATE_LABEL = {
+		CA: translate( 'Province' ),
+	};
+	const STATE_SELECT_TEXT = {
+		CA: translate( 'Select Province' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **53** times:
`translate( 'Province' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).